src/components/IdeaInfoOverlay/reducer.js

```diff
@@ -0,0 +1,28 @@
+import { actions } from "./actions";
+
+const initialState = {
+	open: false,
+	ideaContent: {
+		id: '',
+		name: '',
+		description: '',
+		team: {}
+	}
+};
+
+export default (state = initialState, action) => {
+	switch (action.type) {
+		case actions.open:
+			return {
+				open: true,
+				ideaContent: action.value
+			};
+		case actions.close:
+			return {
+				open: false,
+				ideaContent: initialState.ideaContent
```

Comment:
do you also need the content on close?

Reply:
no I dont, that's why when I close I remove content